docs/install_faq.md

```diff
@@ -104,5 +104,9 @@ I am trying to remove stuff.
 
 **Q: I want to delete my local Helm. Where are all its files?**
 
-Along with the `helm` binary, Helm stores some files in `$HELM_HOME`, which is
-located by default in `~/.helm`.
+Along with the `helm` binary, Helm stores some files in `$HELM_HOME`. Check those
+locations to remove all the configurations too:
+ - `$XDG_CONFIG_DIR/helm` on *nix;
```

Comment:
can we list out what $XDG_CONFIG_DIR is on Linux (`$HOME/.config/helm`)?

Reply:
Also for consistency, can we change *nix to Linux? We refer to "anything other than Windows/MacOS" as linux in other parts of the documentation.

Reply:
$XDG_CONFIG_DIR is different on different unix-based operating system, so it is correct to say this just for linux. And I there are build for different *nix platforms than Linux. Will improve this